sdk/automanage/Azure.ResourceManager.Automanage/samples/Readme.md

```diff
@@ -0,0 +1,129 @@
+# Azure.ResourceManager.Automanager Reference
```

Comment:
Maybe we could give this file a more descriptive name instead of `Readme`?

Reply:
Please check [.NET samples guideline](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples) for naming and other conventions.